netbox_docker_plugin/forms/container.py

```diff
@@ -87,7 +96,14 @@ class ContainerImportForm(NetBoxModelImportForm):
         choices=ContainerStateChoices,
         required=False,
         help_text="Container State. Can be `created`, `restarting`, "
-        + "`running`, `paused`, `exited`or `dead`.",
+        + "`running`, `paused`, `exited `or `dead`.",
```

Comment:
Small typo here:
```suggestion
        + "`running`, `paused`, `exited` or `dead`.",
```

By the way, you don't need the `+` :

```python
"goo" "gle" == "google"
```

Reply:
The `+` is for the respect of pylint ;-)